### ISSUE TITLE
Notice: Use of undefined constant STDERR - assumed 'STDERR' in Akamai\Open\EdgeGrid\Client->getDebugOption() (line 356 of vendor/akamai-open/edgegrid-client/src/Client.php).

### ISSUE THREAD

**Issue by davidbarratt** [akamai/AkamaiOPEN-edgegrid-php-client#24]
I sometimes recieve this error:

```
Notice: Use of undefined constant STDERR - assumed 'STDERR' in Akamai\Open\EdgeGrid\Client->getDebugOption() (line 356 of vendor/akamai-open/edgegrid-client/src/Client.php).
```

**Comment by dshafik** [comment]
@davidbarratt I just released 0.4.4 with this fix, you should be able to `composer update` to grab the latest release, or download the phar file from the [release page](https://github.com/akamai-open/AkamaiOPEN-edgegrid-php/releases/tag/0.4.4)

**Comment by davidbarratt** [comment]
@dshafik,

Thanks!
